ui/src/app/machines/views/MachineList/MachineListTable/MachineListTable.js

```diff
@@ -83,6 +83,25 @@ const getGroupSecondaryString = (machineIDs, selectedIDs) => {
   return string;
 };
 
+/**
+ * Filters columns by hiddenColumns.
+ *
+ * If showActions is true, the "fqdn" column will not be filtered.
```

Comment:
Might be good to add why the fqdn column will not be filtered. e.g. "...will not be filtered because the actions are contained in the fqdn column."